.github/workflows/pr_build_images.yaml

```diff
@@ -30,3 +30,30 @@ jobs:
       - name: Display log on error
         if: failure()
         run: docker logs pulp
+
+  sslbuild:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - ci_type: pulp_ci_centos
+            ci_image: pulp-ci-centos
+            test_type: pulp_galaxy_ng
+            test_image: pulp-galaxy-ng
+          - ci_type: pulp_ci_centos
+            ci_image: pulp-ci-centos
+            test_type: pulp
+            test_image: pulp
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build the images
+        run: |
+          wget https://github.com/just-containers/s6-overlay/releases/download/v1.22.1.0/s6-overlay-amd64.tar.gz
+          docker build --build-arg SCHEME="https" --file ${{ matrix.ci_type }}/Containerfile --tag pulp/${{ matrix.ci_image }}:latest .
```

Comment:
This is subtle.
Does the 'pulp' image use the latest tag for it's base image?

Reply:
Maybe a dumb question, but can we use the `build-arg` to specify the tag of the base image in a Container file?
I have an uneasy feeling with calling the images with the wrong tag to serve the build process.